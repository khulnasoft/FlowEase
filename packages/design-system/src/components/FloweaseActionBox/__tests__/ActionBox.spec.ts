import { render } from '@testing-library/vue';
import FLOWEASEActionBox from '../ActionBox.vue';

describe('FLOWEASEActionBox', () => {
	it('should render correctly', () => {
		const wrapper = render(FLOWEASEActionBox, {
			props: {
				emoji: 'ðŸ˜¿',
				heading: 'Headline you need to know',
				description:
					'Long description that you should know something is the way it is because of how it is. ',
				buttonText: 'Do something',
			},
			global: {
				stubs: ['flowease-heading', 'flowease-text', 'flowease-button', 'flowease-callout'],
			},
		});
		expect(wrapper.html()).toMatchSnapshot();
	});
});
