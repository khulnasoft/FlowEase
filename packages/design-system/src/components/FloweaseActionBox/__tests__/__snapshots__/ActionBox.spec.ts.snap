// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FLOWEASEActionBox > should render correctly 1`] = `
"<div class="flowease-action-box container" data-test-id="action-box">
  <div class="emoji">ğŸ˜¿</div>
  <div class="heading">
    <flowease-heading-stub align="center" tag="span" bold="false" size="xlarge"></flowease-heading-stub>
  </div>
  <div class="description">
    <flowease-text-stub color="text-base" bold="false" size="medium" compact="false" tag="span"></flowease-text-stub>
  </div>
  <flowease-button-stub block="false" element="button" label="Do something" square="false" active="false" disabled="false" loading="false" outline="false" size="large" text="false" type="primary"></flowease-button-stub>
  <!--v-if-->
</div>"
`;
